### PR TITLE
Hotfix: Fix document references from docs.microsoft.com - to learn

### PR DESCRIPTION
**Pull request by ElYusubov**
**Related command**

**Description**

The PR updates documentation references from docs.microsoft.com - to learn.microsoft

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Fix document references

**Comment by jiasli** [comment]
`https://docs.microsoft.com/` automatically redirects to `https://learn.microsoft.com/`:

```
$ curl --include https://docs.microsoft.com/
HTTP/2 301
server: AkamaiGHost
content-length: 0
location: https://learn.microsoft.com/
cache-control: max-age=0
expires: Mon, 13 Feb 2023 07:12:22 GMT
date: Mon, 13 Feb 2023 07:12:22 GMT
set-cookie: original_req_url=https://docs.microsoft.com/; expires=Mon, 13-Feb-2023 07:12:27 GMT; secure; HttpOnly; SameSite=Lax
akamai-cache-status: Redirect from child
report-to: {"group":"network-errors","max_age":604800,"endpoints":[{"url":"https://mdec.nelreports.net/api/report?cat=mdocs"}]}
nel: {"report_to":"network-errors","max_age":604800,"success_fraction":0.01,"failure_fraction":1.0}
```

I am not sure if changing the domain is true necessary. +@dbradish-microsoft

**Review by dbradish-microsoft — APPROVED** [review]

**Comment by dbradish-microsoft** [comment]
@jsntcy, This PR has gone stale and now has merge conflicts.  _The intent of this PR still needs to be completed._  `src/azure-cli/service_name.json` has been fixed, even though these URLs aren't published.  (They are used to validate the selected Azure service.)  The rest of these files are engineering source code.  Somehow this PR has come to the attention of the Core Content PMs and therefore I'm requesting conflict resolution and merge.

**Comment by jsntcy** [comment]
@dbradish-microsoft, is there any impact if we don't apply the changes in this PR?

**Comment by dbradish-microsoft** [comment]
@jsntcy, The URLs will be redirected as @jiasli  stated, but **we'll keep coming up on MSFT reports for non-compliance**.  Eventually, we must change these per MSFT policy.  How about we close this PR, and I'll spin up a new one _without merge conflicts,_ and let's just get it done?  I'll do one for **azure-cli** and another for **azure-cli-extensions.**  

The inconvenience does not outweigh the benefit from my perspective, but I still want to follow the defined rules.  Let's "die on a different hill" so to speak.  :-)

**Comment by dbradish-microsoft** [comment]
@jsntcy , the replacement for this PR is https://github.com/Azure/azure-cli/pull/27040.

File: src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/diagnostic_settings/_create.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "monitor diagnostic-settings create",
 )
 class Create(AAZCommand):
-    """Create diagnostic settings for the specified resource. For more information, visit: https://docs.microsoft.com/rest/api/monitor/diagnosticsettings/createorupdate#metricsettings.
+    """Create diagnostic settings for the specified resource. For more information, visit: https://learn.microsoft.com/rest/api/monitor/diagnosticsettings/createorupdate#metricsettings.
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/diagnostic_settings/_create.py:18]: There is no need to edit files under `aaz` folders as they are automatically generated from swagger https://github.com/Azure/azure-rest-api-specs. They will be overwritten next time we regenerate these files.

**Comment by jiasli** [comment]
Closing as this PR is replaced by https://github.com/Azure/azure-cli/pull/27040.

**Pull request closed without merging**